src/test/java/org/folio/rest/validator/TitlePostBodyValidatorTest.java

```diff
@@ -0,0 +1,246 @@
+    titlePostRequest
+      .withData(new TitlePostData()
+        .withAttributes(new TitlePostDataAttributes()
+          .withName(TITLE_TEST_NAME)
+          .withPublicationType(org.folio.rest.jaxrs.model.PublicationType.BOOK)
+          .withDescription("")))
+      .withIncluded(getTitlePostIncluded());
+
+    validator.validate(titlePostRequest);
+  }
+  @Test
+  public void shouldNotThrowExceptionIfTitleDescriptionIsNull(){
+    TitlePostRequest titlePostRequest = new TitlePostRequest();
+    titlePostRequest
+      .withData(new TitlePostData()
+        .withAttributes(new TitlePostDataAttributes()
+          .withName(TITLE_TEST_NAME)
+          .withPublicationType(org.folio.rest.jaxrs.model.PublicationType.BOOK)
+          .withDescription(null)))
+      .withIncluded(getTitlePostIncluded());
+
+    validator.validate(titlePostRequest);
+  }
+
+  @Test
+  public void shouldNotThrowExceptionIfTitleEditionIsEmpty(){
+    TitlePostRequest titlePostRequest = new TitlePostRequest();
+    titlePostRequest
+      .withData(new TitlePostData()
+        .withAttributes(new TitlePostDataAttributes()
+          .withName(TITLE_TEST_NAME)
+          .withPublicationType(org.folio.rest.jaxrs.model.PublicationType.BOOK)
+          .withEdition("")))
+      .withIncluded(getTitlePostIncluded());
+
+    validator.validate(titlePostRequest);
+  }
+
+  @Test
+  public void shouldNotThrowExceptionIfTitleEditionIsNull(){
+    TitlePostRequest titlePostRequest = new TitlePostRequest();
+    titlePostRequest
+      .withData(new TitlePostData()
+        .withAttributes(new TitlePostDataAttributes()
+          .withName(TITLE_TEST_NAME)
+          .withPublicationType(org.folio.rest.jaxrs.model.PublicationType.BOOK)
+          .withEdition(null)))
+      .withIncluded(getTitlePostIncluded());
+
+    validator.validate(titlePostRequest);
+  }
+
+  @Test
+  public void shouldNotThrowExceptionIfTitlePublisherNameIsNull(){
+    TitlePostRequest titlePostRequest = new TitlePostRequest();
+    titlePostRequest
+      .withData(new TitlePostData()
+        .withAttributes(new TitlePostDataAttributes()
+          .withName(TITLE_TEST_NAME)
+          .withPublicationType(org.folio.rest.jaxrs.model.PublicationType.BOOK)
+          .withPublisherName(null)))
+      .withIncluded(getTitlePostIncluded());
+
+    validator.validate(titlePostRequest);
+  }
+
+  @Test
+  public void shouldNotThrowExceptionIfTitlePublisherNameIsEmpty(){
+    TitlePostRequest titlePostRequest = new TitlePostRequest();
+    titlePostRequest
+      .withData(new TitlePostData()
+        .withAttributes(new TitlePostDataAttributes()
+          .withName(TITLE_TEST_NAME)
+          .withPublicationType(org.folio.rest.jaxrs.model.PublicationType.BOOK)
+          .withPublisherName("")))
+      .withIncluded(getTitlePostIncluded());
+
+    validator.validate(titlePostRequest);
+  }
+
+  private java.util.List<org.folio.rest.jaxrs.model.TitlePostIncluded> getTitlePostIncluded() {
```

Comment:
Very minor issue, this PR has a lot of fully qualified class names, for example org.folio.rest.jaxrs.model.TitlePostIncluded